forte/data/data_store.py

```diff
@@ -516,12 +564,17 @@ def _add_entry_raw(
             except KeyError:
                 self.__elements[type_name] = SortedList(key=sorting_fn)
                 self.__elements[type_name].add(entry)
-        elif entry_type in [Link, Group]:
+        elif entry_type in [Link, Group, Generics]:
             try:
                 self.__elements[type_name].append(entry)
             except KeyError:
                 self.__elements[type_name] = []
                 self.__elements[type_name].append(entry)
+        else:
+            raise NotImplementedError(
```

Comment:
We can add this error to `Raises:` section in docstring

Reply:
Added it.